nodeJS/express/installing_postgresql.md

```diff
@@ -0,0 +1,293 @@
+<!-- markdownlint-disable MD024 TOP004 -->
```

Comment:
I feel like we shouldn't disable rules as a temporary workaround to avoiding check failures, but rather for when we actually intend to disable a rule for good for whatever reason. I'd personally rather merge something that we know fails the linter, but intend to resolve as a followup. @MaoShizhong wdyt?

Reply:
Perhaps follow the Rails version with a todo comment? Not the most ideal but it's at least consistent 

```suggestion
<!-- TODO: Revisit lesson/heading structure to remove need to disable rules -->
<!-- markdownlint-disable MD024 TOP004 -->
```

Reply:
That sounds good

Reply:
above change made.